examples/00_vanilla/vanilla_example.py

```diff
@@ -0,0 +1,136 @@
+# Required for rendering initialization, not necessary for
+# local rendering, but doesn't hurt to include it
+from numpy import asarray
+from pathlib import Path
+from PIL import Image
+from trame.app.testing import enable_testing
+from trame.decorators import TrameApp, change
+from trame_rca.widgets import rca
+from trame_rca.utils import AbstractRenderWindow
+from trame.app import get_server
+from trame.ui.vuetify3 import SinglePageLayout
+from trame.widgets import vuetify3 as v3
+
+
+DEFAULT_ROTATION_STEP = 45
+
+
+class ImageRenderWindow(AbstractRenderWindow):
+    """
+    A concrete implementation of AbstractRenderWindow.
+    In order to function correctly, the following abstract methods must be overridden:
+
+    1. ` img_cols_rows(self) -> tuple[NDArray, int, int]`
```

Comment:
NIP : Comment is not very helpful IMO since the AbstractRenderWindow is marked ABC it will fail to instantiate if every abstract method is not implemented. 

This comment may be duplicated from the AbstractRenderWindow and will probably be outdated when the parent class is updated.